### PR TITLE
Add storage driver requirements to EKS install guide

### PR DESCRIPTION
**Pull request by codyjlandstrom**
Add a step to the EKS SH install guide to add the Amazon EBS CLI storage driver add-on

### CODE REVIEW

**Review by rberrelleza — APPROVED** [review]

File: src/content/get-started/cloud-provider-guides/eks.mdx [hunk 1]
```diff
@@ -54,6 +54,14 @@ helm repo add okteto https://charts.okteto.com
 helm repo update
 ```
 
+### Configure the Amazon EBS CSI storage driver
```
Comment (pchico83) [inline src/content/get-started/cloud-provider-guides/eks.mdx:57]: If we assume the storage driver is available, the user can use this configuration from the beginning:
```
buildkit:
  persistence:
    enabled: true

registry:
  storage:
    filesystem:
      persistence:
        enabled: true
```
Moreover, should we default to persistence for buildkit and the registry in our helm values file?

Comment (codyjlandstrom) [reply src/content/get-started/cloud-provider-guides/eks.mdx:57]: < Moreover, should we default to persistence for buildkit and the registry in our helm values file?
That makes sense to me. Should we call that out specifically here or just add it to the example file?

Comment (pchico83) [reply src/content/get-started/cloud-provider-guides/eks.mdx:57]: @codyjlandstrom I think there are two action items here:
- Change the helm values file in this guide as part of this PR. [here](https://github.com/okteto/docs/blob/main/src/content/self-hosted/eks/config.yaml) and [here](https://github.com/okteto/docs/blob/main/src/content/get-started/cloud-provider-guides/eks.mdx#L74)
- Create a jira ticket to update our helm values default file in the chart, which would apply to all the installations. That will enable these values by default across all installations:
```
buildkit:
  persistence:
    enabled: true

registry:
  storage:
    filesystem:
      persistence:
        enabled: true
```
and we could remove them from all the guides. If we do that, we should return a clear error when running `helm install` and a storage driver is unavailable. Maybe with a helm pre-install hook. And remove the step to configure buildkit persistency from the Okteto UI installation dashboard

Comment (codyjlandstrom) [reply src/content/get-started/cloud-provider-guides/eks.mdx:57]: @pchico83 Updated the PR and created a Jira ticket here: https://okteto.atlassian.net/browse/RAIN-131

**Review by pchico83 — APPROVED** [review]

File: src/content/get-started/cloud-provider-guides/eks.mdx (hunk 1 above)
Comment (pchico83) [reply src/content/get-started/cloud-provider-guides/eks.mdx:57]: Thanks @codyjlandstrom!

**Pull request merged**